### PR TITLE
Warn if modules passed to pm-uninstall are not enabled

### PR DESCRIPTION
**Pull request by DieterHolvoet**
Now it prints _'Successfully uninstalled: \<moduleName\>'_ which is confusing IMO.

### CODE REVIEW

File: src/Drupal/Commands/pm/PmCommands.php [hunk 1]
```diff
@@ -169,7 +169,18 @@ public function validateEnableModules(CommandData $commandData): void
     public function uninstall(array $modules): void
     {
         $modules = StringUtils::csvToArray($modules);
-        $list = $this->addUninstallDependencies($modules);
+
+        $enabled_modules = array_filter($modules, function ($module) {
+            return $this->getModuleHandler()->moduleExists($module);
+        });
+        if ($enabled_modules === []) {
+            throw new \Exception(dt('The following module(s) are not enabled: !list. No modules to uninstall.', ['!list' => implode(', ', $modules)]));
```
Comment (weitzman) [inline src/Drupal/Commands/pm/PmCommands.php]: I think the right language now is not enabled/disabled but installed/uninstalled. We just did a PR to change the from:enable to pm:install

**Review by weitzman — CHANGES_REQUESTED** [review]
LGTM. Minor change requested.

File: src/Drupal/Commands/pm/PmCommands.php (hunk 1 above)
Comment (DieterHolvoet) [reply src/Drupal/Commands/pm/PmCommands.php]: Changed

**Comment by weitzman** [comment]
I'm wondering this this will break CI or scripts for anyone. I think it will, with the new Exception. We should make a new minor release, and not a patch release.

**Review by weitzman — APPROVED** [review]
Approved. Lets merge before we release a new minor. That way this doesn't accidentally go out in a patch release.

**Pull request merged**